recipe/meta.yaml

```diff
@@ -7,40 +7,37 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 89c022dad41d28e26a1290eb1585db1042ccb09e6951220bb5c9146e2f795147
+  sha256: a9096caa76c3705be7ce5a14e6881bb8e8c2cd097da4b2d9376260f584f36be5
 
 build:
   number: 0
-  skip: true  # [py<36]
+  skip: true  # [py<38]
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - poetry
   host:
     - python
     - pip
-    - typing_extensions >=3.7.4.3  # [py<=38]
+    - poetry
+    - typing_extensions >=4.2  # [py>=38]
   run:
     - python
-    - typing_extensions >=3.7.4.3  # [py<=38]
+    - poetry
+    - typing_extensions >=4.2  # [py>=38]
 
-test:
```

Comment:
Why were the test commands removed?

Reply:
The tests you had were not right for the way we are installing `pandas-stubs`, which is as a separate package.  Trying to figure out what kind of test to use.